cloud-core/src/test/java/cloud/commandframework/CommandSuggestionsTest.java

```diff
@@ -390,7 +404,7 @@ void testLiteralWithVariable() {
         Assertions.assertEquals(Arrays.asList("vici", "vidi"), suggestions3);
         final String input4 = "literal_with_variable vidi";
         final List<String> suggestions4 = manager.suggest(new TestCommandSender(), input4);
-        Assertions.assertEquals(Collections.emptyList(), suggestions4);
+        Assertions.assertEquals(Collections.singletonList("vidi"), suggestions4);
```

Comment:
This test was disabled & wrong previously. Without my change the test wouldn't pass as well as "vidi" would be given as a suggestion.